src/crypto.rs

```diff
@@ -42,6 +42,7 @@ pub fn do_hmac(
  * PBKDF2 function defaults to SHA-1 unless otherwise specified, and
  * Python-Keylime uses this default.
  */
+#[allow(dead_code)]
```

Comment:
Please don't add these.

Reply:
Without this would cause function unused warning if that is not an issue.

Reply:
Got it. Thanks.